Editor/Editors/CinemachineVirtualCameraBaseEditor.cs

```diff
@@ -111,21 +118,37 @@ protected void DrawExtensionsWidgetInInspector()
                 EditorGUILayout.Space();
                 EditorGUILayout.LabelField("Extensions", EditorStyles.boldLabel);
                 Rect rect = EditorGUILayout.GetControlRect(true, EditorGUIUtility.singleLineHeight);
-                rect = EditorGUI.PrefixLabel(rect, new GUIContent("Add Extension"));
+                rect = EditorGUI.PrefixLabel(rect, Styles.addExtensionLabel);
 
                 int selection = EditorGUI.Popup(rect, 0, sExtensionNames);
                 if (selection > 0)
                 {
                     Type extType = sExtensionTypes[selection];
-                    if (Target.GetComponent(extType) == null)
-                        Undo.AddComponent(Target.gameObject, extType);
+
+                    if (Selection.objects.Length == 1)
```

Comment:
Why this needs separate code paths for 1 vs non-1 length? From a quick look it feels like the "loop over targets" would just work fine for 1 length as well?

Reply:
I missed that when refactoring, don't think there's any other reason to it.